soc/arm/nordic_nrf/nrf53/Kconfig.soc

```diff
@@ -47,6 +49,25 @@ config SOC_NRF53_ANOMALY_160_WORKAROUND
 	depends on SYS_CLOCK_EXISTS
 	select ARM_ON_ENTER_CPU_IDLE_HOOK
 
+config SOC_NRF53_RTC_PRETICK
+	bool
+	depends on !NRF_802154_RADIO_DRIVER
```

Comment:
Once https://github.com/nrfconnect/sdk-nrf/pull/12316 moves one CC from from nrf_rtc_timer.c to RTC0, get rid of 
```
depends on !NRF_802154_RADIO_DRIVER
```

Reply:
done